integrated_channels/degreed2/client.py

```diff
@@ -303,6 +335,36 @@ def update_content_metadata(self, serialized_data):
             patch_url,
             course_id
         )
+        # once course is updated successfully, we need to do 2 more steps
+        # 1. Fetch skills from enterprise-catalog
+        client = EnterpriseCatalogApiClient()
```

Comment:
wouldn't it be better to keep this client in the Degreed2 initialization? Otherwise it will create a new client for every course create and update call.